02_bubble/lokili/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func bubble(arr []int) []int {
+	arrLen := len(arr)
+	sorted := make([]int, arrLen)
+	copy(sorted, arr)
+
+	for i := 0; i < arrLen-1; i++ {
+		swapped := false
+		for j := 0; j < arrLen-i-1; j++ {
```

Comment:
Counting down with `i` simplifies the `j` for-loop:
```go
	for i := arrLen-1; i > 0; i++ {
		swapped := false
		for j := 0; j < i; j++ {
```

Reply:
I've changed it to the right format.